server/src/controller/userController.js

```diff
@@ -20,6 +20,33 @@ export const get = async (req, res) => {
   return res.status(StatusCodes.ACCEPTED).json({ users });
 };
 
+export const getUserWhoWorkedTheMostHours = async (req, res) => {
+  if (validate(req, res)) {
+    return res;
+  }
+  const users = await prisma.user.findMany({
+    where: {
```

Comment:
you should instead use an aggregation query, and group the shifts by user id.

Reply:
the way you did it works, this is just a more efficient method and is good to learn

Reply:
i gave the prisma.user.aggregate() method a shot, the docs were a little confusing, need some help
https://www.prisma.io/docs/concepts/components/prisma-client/aggregation-grouping-summarizing

Reply:
const users = await prisma.user.aggregate({
    where: {
      shiftsWorked: {
        isEmpty: false, // we only want users who have worked at least one shift
      },
    },
    orderBy: {
      _sum: {
        shiftsWorked: {
          end: { decrement: start },
        },
      },
      _sum: "asc",
    },
  });


I'm assuming this is wrong ^

Reply:
got it, we can work on it monday